kubernetes/monitoring/cron-hdfs/cmsmon-rucio-ds.yaml

```diff
@@ -0,0 +1,113 @@
+# cmsmon-cmsmon-rucio-ds CronJob for K8s
+kind: Service
+apiVersion: v1
+metadata:
+  name: cmsmon-rucio-ds
+  namespace: cron-hdfs
+spec:
+  selector:
+    app: cmsmon-rucio-ds
+  type: NodePort
+  ports:
+    - name: port-0 # spark.driver.port, cern default is 5201
+      nodePort: 32501
+      port: 32501
+      protocol: TCP
+      targetPort: 32501
+    - name: port-1 # spark.driver.blockManager.port
+      nodePort: 32502
+      port: 32502
+      protocol: TCP
+      targetPort: 32502
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cmsmon-rucio-ds
+  namespace: cron-hdfs
+  labels:
+    app: cmsmon-rucio-ds
+# Reference: https://www.ibm.com/support/pages/aix-cron-cron-environment-and-cron-job-failures
```

Comment:
We can add a `README.md` file to `cron-hdfs` folder and put this reference and other explanations there. What do you think?